ff4s/src/main/scala/ff4s/CancellationKey.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 buntec
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ff4s
+
+import cats.Eq
+import cats.Show
+
+final case class CancellationKey(key: String)
```

Comment:
I wonder if instead of `String` it should be a `Unique.Token`?

Reply:
How about this?